.bazelrc

```diff
@@ -14,6 +14,9 @@ build:linux --cxxopt=-std=c++20
 build:macos --cxxopt=-std=c++20 --cpu=darwin_x86_64
 build:windows --cxxopt=/std:c++20 --cxxopt=/Zc:preprocessor
 
+# this requires developer mode, but is required to have pack installer functioning
+common:windows --windows_enable_symlinks --enable_runfiles
```

Comment:
Iirc, the first option needs to be passed (unconditionally) as startup option, unless something changed when I introduced this in the internal repo.